src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,22 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import { UpdatedTodo } from '../../types/UpdatedTodo';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: UpdatedTodo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id="1"
+      className={`TodoInfo ${classNames({
+        'TodoInfo--completed': todo.completed,
+      })}`}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo user={todo.user} />
```

Comment:
and just add the check here
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```